doc/api/modules.md

```diff
@@ -425,6 +426,30 @@ These are mostly for historic reasons.  **You are highly encouraged
 to place your dependencies locally in `node_modules` folders.**  They
 will be loaded faster, and more reliably.
 
+## The module wrapper
+
+<!-- type=misc -->
+
+Before Node.js executes the code in your module, it wraps it with a function
+wrapper that looks like this:
```

Comment:
I know this particular document is riddled with `you` and `your`, which we should get refactored out later.. can you reword this to avoid adding a new instance of `your`? Perhaps, `Before a module's code is executed, Node.js will wrap it with a function wrapper.`